embedded/src/main/java/org/wildfly/core/embedded/EmbeddedStandaloneServerFactory.java

```diff
@@ -336,10 +336,10 @@ private void exit() {
                 } catch (RuntimeException rte) {
                     throw rte;
                 } catch (InterruptedException ite) {
-                    ite.printStackTrace();
+                    ServerLogger.ROOT_LOGGER.error(ite.getLocalizedMessage(), ite);
```

Comment:
@luck3y I believe these fixes in this file should all use EmbeddedLogger.ROOT_LOGGER, and the uses of ServerLogger in EmbeddedHostControllerFactory should switch to using EmbeddedLogger. (AFAICT the ServerLogger messages used there are not used outside that class so they can just move.) The EmbeddedHostControllerFactory change being a task for you, not for this PR. WDYT?

Reply:
@bstansberry I'm of course OK changing to the EmbeddedLogger. i've poundered about for a little while, not knowing what was best here...

Reply:
@rpelisse ok, let's go with EmbeddedLogger. Thanks!